packages/repositories.config

```diff
@@ -1,5 +1,10 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <repositories>
+  <repository path="../src/NUnitFramework/framework/packages.config" />
+  <repository path="../src/NUnitFramework/nunitlite.tests/packages.config" />
+  <repository path="../src/NUnitFramework/nunitlite/packages.config" />
+  <repository path="../src/NUnitFramework/testdata/packages.config" />
+  <repository path="../src/NUnitFramework/tests/packages.config" />
```

Comment:
Ahh, so it comes from MonoDevelop! Does MonoDevelop auto-regenerate it? Could we .gitignore it instead?